src/components/Table/TableHeader.vue

```diff
@@ -0,0 +1,15 @@
+
+<script>
+import { h } from 'vue';
+export default {
+  name: 'TableHeader',
+  render () {
```

Comment:
Ohhhh so fancy! ✨  Will you talk about render functions in our next in-person standup? This is the first one we used and it'd be interesting to hear why you chose it.